src/main/scala/nl.knaw.dans.easy.deposit/EasyDepositApiApp.scala

```diff
@@ -63,13 +63,14 @@ class EasyDepositApiApp(configuration: Configuration) extends DebugEnhancedLoggi
     val dir = getConfiguredDirectory("deposits.stage-zips")
     logger.info(s"Uploads are staged in $dir")
     if (dir.nonEmpty) {
-      val msg = s"Pending uploads were probably interrupted. See logs lines with 'POST /deposit/{id}/file/{dir_path}'. Please remove their directories from the staging area: $dir"
-      logger.error(msg)
-      throw new FileAlreadyExistsException(msg)
+      throw new FileAlreadyExistsException(
+        s"Possibly pending or interrupted uploads in the staging area: $dir. See log lines with 'POST /deposit/{id}/file/{dir_path}'."
```

Comment:
The sentence starting with `See log lines` doesn't make any sense to me, yet.

Reply:
The directory should be empty after completing request. The most important reason for left overs could be force shutdown during upload. Logged request without response can conform that situation. Otherwise we might have other problems.